packages/block-editor/src/hooks/use-editor-wrapper-styles.native.js

```diff
@@ -0,0 +1,250 @@
+/**
+ * External dependencies
+ */
+import { useWindowDimensions } from 'react-native';
+
+/**
+ * WordPress dependencies
+ */
+import { useMemo } from '@wordpress/element';
+import {
+	ALIGNMENT_BREAKPOINTS,
+	WIDE_ALIGNMENTS,
+	alignmentHelpers,
+} from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import styles from './use-editor-wrapper-styles.scss';
+
+/**
+ * Get the styles for the wide width alignment.
+ *
+ * @param {string}  align       The alignment value.
+ * @param {boolean} isLandscape Whether the screen is in landscape mode.
+ * @param {number}  width       The width of the screen.
+ * @return {Object} An object containing the styles for the wide width alignment.
+ */
+export function getWideWidthStyles( align, isLandscape, width ) {
+	if ( align !== WIDE_ALIGNMENTS.alignments.wide ) {
+		return {};
+	}
+
+	if ( isLandscape && width < ALIGNMENT_BREAKPOINTS.large ) {
+		return styles[
+			'block-editor-hooks__use-editor-wrapper-styles-alignment--wide-landscape'
+		];
+	}
+
+	if ( width <= ALIGNMENT_BREAKPOINTS.small ) {
+		return { maxWidth: width };
+	}
+
+	if (
+		width >= ALIGNMENT_BREAKPOINTS.medium &&
+		width < ALIGNMENT_BREAKPOINTS.wide
+	) {
+		return styles[
+			'block-editor-hooks__use-editor-wrapper-styles-alignment--wide-medium'
+		];
+	}
+
+	return styles[
+		'block-editor-hooks__use-editor-wrapper-styles-alignment--wide'
+	];
+}
+
+/**
+ * Get the styles for the full width alignment.
+ *
+ * @param {string}  align           The alignment value.
+ * @param {string}  blockName       The name of the block.
+ * @param {boolean} hasParents      Whether the block has parents.
+ * @param {string}  parentBlockName The name of the parent block.
+ * @return {Object} An object containing the styles for the full width alignment.
+ */
+export function getFullWidthStyles(
+	align,
+	blockName,
+	hasParents,
+	parentBlockName
+) {
+	const { isContainerRelated, isFullWidth } = alignmentHelpers;
+	const fullWidthStyles = isFullWidth( align )
+		? styles[
+				'block-editor-hooks__use-editor-wrapper-styles-alignment--full'
+		  ]
+		: {};
+
+	if (
+		! align &&
+		hasParents &&
+		! isContainerRelated( parentBlockName ) &&
+		isContainerRelated( blockName )
+	) {
+		fullWidthStyles.paddingHorizontal =
+			styles[
+				'block-editor-hooks__use-editor-wrapper-styles-block'
+			].marginLeft;
```

Comment:
It appears this selector (and many others referenced in this file) only contain a single attribute. Do we see value in using single-attribute styles (Sass) rather than holding the values in the JavaScript file?

Reply:
Updated in https://github.com/WordPress/gutenberg/pull/49616/commits/696d5a97dabea9f96168ccdf80af6b2c8491df3b#diff-48fae6ad7c03074aba0381f663a92c2882e22ef6504fe8ae49a4b49fa5f0161f